apps/web/pages/[user]/[type].tsx

```diff
@@ -112,13 +146,14 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     },
   });
 
-  if (!user) {
+  if (!users) {
     return {
       notFound: true,
     };
   }
+  const [user] = users; //to be used when dealing with single user, not dynamic group
 
-  if (user.eventTypes.length !== 1) {
+  if (users.length === 1 && user.eventTypes.length !== 1) {
```

Comment:
can we move this constant comparison to a named const?
`const isSingleUser = users.length === 1;`
`const isDynamicGroup = users.length > 1;`